rusk/src/bin/config.rs

```diff
@@ -31,25 +31,21 @@ const DEFAULT_LOG_TYPE: &str = "coloured";
 
 impl From<&ArgMatches> for Config {
     fn from(matches: &ArgMatches) -> Self {
-        let mut rusk_config =
-            matches
-                .value_of("config")
-                .map_or(Config::default(), |conf_path| {
-                    let toml = std::fs::read_to_string(conf_path).unwrap();
-                    toml::from_str(&toml).unwrap()
-                });
+        let mut rusk_config = matches.get_one::<String>("config").map_or(
+            Config::default(),
+            |conf_path| {
+                let toml = std::fs::read_to_string(conf_path).unwrap();
+                toml::from_str(&toml).unwrap()
+            },
+        );
 
-        rusk_config.kadcast_test = matches.is_present("kadcast_test");
+        rusk_config.kadcast_test = matches.contains_id("kadcast_test");
 
         // Overwrite config log-level
-        if let Some(log_level) = matches.value_of("log-level") {
-            rusk_config.log_level = Some(log_level.into());
-        }
+        rusk_config.log_level = matches.get_one::<String>("log-level").cloned();
 
         // Overwrite config log-type
-        if let Some(log_type) = matches.value_of("log-type") {
-            rusk_config.log_type = Some(log_type.into());
-        }
+        rusk_config.log_type = matches.get_one::<String>("log-type").cloned();
```

Comment:
The `if let` statement was used in order to have a default loglevel.
Indeed, if not log-level args were set, the default "INFO" were used.

With these changes I suppose that `None` is set if no args are specified